examples/objectServerExample/src/main/java/io/realm/examples/objectserver/CounterActivity.java

```diff
@@ -86,12 +121,21 @@ public void onChange(CRDTCounter counter) {
                 }
             });
             counterView.setText("0");
+
+            // Setup progress listeners for indeterminate progress bars
+            session = SyncManager.getSession(config);
+            session.addDownloadProgressListener(ProgressMode.INDEFINITELY, downloadListener);
+            session.addUploadProgressListener(ProgressMode.INDEFINITELY, uploadListener);
         }
     }
 
     @Override
     protected void onStop() {
         super.onStop();
+        if (session != null) {
+            session.removeProgressListener(downloadListener);
```

Comment:
It isn't. GitHub is just folding it weirdly. If you see the full source you can see it is `onStart`